### ISSUE TITLE
Cordova samples need the Whitelist plugin for XHR requests

### ISSUE THREAD

**Issue by ggailey777** [Microsoft/cordova-samples#11]
It looks like you now need the Whitelist plugin to be able to make remote xhr requests to Mobile Services (or any remote URL), which is what the client does. I submitted a PR with the fix #10, but you probably will need the same fix in other samples that access mobile services or any other remote service using XHR, which is why I also opened this issue.

**Comment by NicoJuicy** [comment]
I'm not sure, but if you look at plugin.xml there is a whitelist there. Isn't that the same thing?

**Comment by ggailey777** [comment]
You are right that the whitelist plugin is not needed, but Android requires a Content-Security-Policy be set to allow xhr.
